examples/plot_depth_classification.py

```diff
@@ -210,10 +209,8 @@ def _plot_boundaries(axis):
 # | NearestClass | DDGClassifier with nearest neighbors |
 # +--------------+--------------------------------------+
 
-ddg: DDGTransformer[FDataGrid] = DDGTransformer(
-    depth_method=ModifiedBandDepth(),
-)
-X_train_trans = ddg.fit_transform(X_train, y_train)
+pct = PerClassTransformer(ModifiedBandDepth())
```

Comment:
You should specify `array_output=True`, otherwise it fails at line 225 (see docs output).